root/static/scripts/tests/react-macros.js

```diff
@@ -0,0 +1,155 @@
+// This file is part of MusicBrainz, the open internet music database.
```

Comment:
This evals the React snippets from the Perl tests and compares them to the corresponding TT output. Comparing two HTML strings is far, far easier in JavaScript than in it is Perl (the DOM has a built-in HTML parser).